dtformats/indx_directory_entry.yml

```diff
@@ -0,0 +1,158 @@
+name: index_record
+type: format
+description: Index Directory Entry
```

Comment:
(change to something in line with) NTFS $I30 index, which contains directory entries